lapack/testlapack/dlassq.go

```diff
@@ -72,15 +72,7 @@ func dlassqTest(t *testing.T, impl Dlassqer, rnd *rand.Rand, n, incx, cas int, v
 		workssq += wi * wi
 	}
 
-	// Construct a test (n+1)-vector z as
```

Comment:
The diff is ugly but I only simplified the test by getting rid of slice `z` and moved the computation of `nrmWant` closer to the check.